### ISSUE TITLE
ERROR in ./~/react-tap-event-plugin/src/injectTapEventPlugin.js

### ISSUE THREAD

**Issue by ccfiel** [dternyak/React-Redux-Flask#7]
I got this error. Any idea how to resolve this?

```
ERROR in ./~/react-tap-event-plugin/src/injectTapEventPlugin.js
Module not found: Error: Cannot resolve module 'react/lib/EventPluginHub' in /home/chris/Projects/Web/tailpos/static/node_modules/react-tap-event-plugin/src
 @ ./~/react-tap-event-plugin/src/injectTapEventPlugin.js 23:2-37

ERROR in ./~/react-tap-event-plugin/src/TapEventPlugin.js
Module not found: Error: Cannot resolve module 'react/lib/EventConstants' in /home/chris/Projects/Web/tailpos/static/node_modules/react-tap-event-plugin/src
 @ ./~/react-tap-event-plugin/src/TapEventPlugin.js 22:21-56

ERROR in ./~/react-tap-event-plugin/src/TapEventPlugin.js
Module not found: Error: Cannot resolve module 'react/lib/EventPluginUtils' in /home/chris/Projects/Web/tailpos/static/node_modules/react-tap-event-plugin/src
 @ ./~/react-tap-event-plugin/src/TapEventPlugin.js 23:23-60

ERROR in ./~/react-tap-event-plugin/src/TapEventPlugin.js
Module not found: Error: Cannot resolve module 'react/lib/EventPropagators' in /home/chris/Projects/Web/tailpos/static/node_modules/react-tap-event-plugin/src
 @ ./~/react-tap-event-plugin/src/TapEventPlugin.js 24:23-60

ERROR in ./~/react-tap-event-plugin/src/TapEventPlugin.js
Module not found: Error: Cannot resolve module 'react/lib/SyntheticUIEvent' in /home/chris/Projects/Web/tailpos/static/node_modules/react-tap-event-plugin/src
 @ ./~/react-tap-event-plugin/src/TapEventPlugin.js 25:23-60

ERROR in ./~/react-tap-event-plugin/src/TapEventPlugin.js
Module not found: Error: Cannot resolve module 'react/lib/ViewportMetrics' in /home/chris/Projects/Web/tailpos/static/node_modules/react-tap-event-plugin/src
 @ ./~/react-tap-event-plugin/src/TapEventPlugin.js 27:22-58
```

Maybe this is related? https://github.com/callemall/material-ui/issues/5573

**Comment by j138** [comment]
freeze react version.
after `rm -rf node_modules && npm install`

```
% cat package.json | grep react
      react: ~15.3.0,
      react-addons-create-fragment: ~15.3.0,
      react-addons-css-transition-group: ~15.3.0,
      react-addons-pure-render-mixin: ~15.3.0,
      react-addons-transition-group: ~15.3.0,
      react-addons-shallow-compare: ~15.3.0,
      react-addons-update: ~15.3.0,
      react-dom: ~15.3.0,
```

**Comment by ccfiel** [comment]
@j138 Thanks for the reply. How can I freeze react version. I am new in this thing. :(. I tried this 

```
npm install -g npm-freeze
npm-freeze manifest
rm -rf node_modules
npm install
npm-freeze check
```

but still error persist. I notice your react version is 15.3.0 but package.json in the code is 15.3.1? Do I have to change the package.json?

```
chris@chris:~/React-Redux-Flask/static$ cat package.json | grep react
    "react",
    "reactjs",
    "babel-plugin-react-transform": "^2.0.0",
    "babel-preset-react": "^6.3.13",
    "babel-preset-react-hmre": "^1.0.1",
    "eslint-plugin-react": "^6.1.2",
    "react": "^15.3.1",
    "react-addons-css-transition-group": "^15.3.1",
    "react-calendar-component": "^1.0.0",
    "react-date-picker": "^5.3.28",
    "react-datepicker": "^0.29.0",
    "react-document-meta": "^2.0.0-rc2",
    "react-dom": "^15.1.0",
    "react-forms": "^2.0.0-beta33",
    "react-hot-loader": "^1.3.0",
    "react-loading-order-with-animation": "^1.0.0",
    "react-onclickoutside": "^5.3.3",
    "react-redux": "^4.3.0",
    "react-router": "2.7.0",
    "react-router-redux": "^4.0.0",
    "react-tap-event-plugin": "^1.0.0",
    "react-transform-hmr": "^1.0.1",
```

**Comment by j138** [comment]
I freezed react@15.3 that resolved problem.
`s/^15.3/~15.3/g`

BTW, resolved this issue.
callemall/material-ui#5573

I just fixed after update react@15.4 and react-tap-event-plugin@2.0.1.
